tests/unit/type/string.tcl

```diff
@@ -394,6 +394,25 @@ start_server {tags {"string"}} {
         list $v1 $v2 [r get foo]
     } {{} OK 2}
 
+    test {Extended SET GET option} {
+        r del foo
+        r set foo bar
+        r set foo bar2 GET
+    } {bar}
+
+    test {Extended SET GET option with no previous value} {
+        r del foo
+        r set foo bar GET
+    } {}
```

Comment:
let's extend these two tests with another lines that verifies the new value was indeed set.